dotCMS/src/main/java/com/dotcms/publisher/receiver/BundlePublisher.java

```diff
@@ -202,7 +202,14 @@ public PublisherConfig process ( final PublishStatus status ) throws DotPublishi
 
             //Get the identifiers on this bundle
             assetsDetails = new HashMap<>();
-            bundlerAssets = (List<PublishQueueElement>)((HashMap)BundlerUtil.xmlToObject(xml)).get("ASSETS");
+
+            final Object rawObject = BundlerUtil.xmlToObject(xml);
+
+            if (rawObject != null && Map.class.isInstance(rawObject)) {
+                final Map bundleMap = (Map) rawObject;
+                bundlerAssets = bundleMap.get("ASSETS") != null ?
```

Comment:
This check is unnecessary. Just assign whatever `bundleMap.get("ASSETS")` retrieves.

Reply:
If I do that way then I am going to get a ClassCastException here: (List<PublishQueueElement>) bundleMap.get("ASSETS")

Reply:
No you don't. This will be compiled ok:
```java
List<String> l = (List<String>) null;
```